.github/workflows/bench.yml

```diff
@@ -33,10 +33,16 @@ jobs:
 
       # The compression benchmarks rely on DuckDB being installed to convert CSV to Parquet
       - name: Install DuckDB
-        uses: opt-nc/setup-duckdb-action@v1.0.9
+        uses: adamgs/setup-duckdb-action@c9a7a95cba10a46746b8477c22e4ca3fce77e164
```

Comment:
if we need a fork, let's fork this action into our org (or move the fork)

Reply:
just make it a curl and untar, duckdb builds standalone executables

Reply:
I'll move the fork, I think that's the easiest thing here